src/include/daos/mgmt.h

```diff
@@ -72,5 +72,5 @@ int dc_mgmt_sys_attach(const char *name, struct dc_mgmt_sys **sysp);
 void dc_mgmt_sys_detach(struct dc_mgmt_sys *sys);
 ssize_t dc_mgmt_sys_encode(struct dc_mgmt_sys *sys, void *buf, size_t cap);
 ssize_t dc_mgmt_sys_decode(void *buf, size_t len, struct dc_mgmt_sys **sysp);
-
+int dc_mgmt_disconnect();
```

Comment:
(style)  Bad function definition - int dc_mgmt_disconnect() should probably be int dc_mgmt_disconnect(void)